lib/strong_migrations/checker.rb

```diff
@@ -192,7 +192,34 @@ def postgresql?
     end
 
     def postgresql_version
-      @postgresql_version ||= connection.execute("SHOW server_version_num").first["server_version_num"].to_i
+      @postgresql_version ||= target_version || connection.execute("SHOW server_version_num").first["server_version_num"].to_i
+    end
+
+    def rails_test_or_development?
+      if defined?(Rails)
+        Rails.env.test? || Rails.env.development?
+      else
+        false
+      end
+    end
+
+    def target_version
```

Comment:
Think we can simplify this method significantly. Basically, just need:

```ruby
StrongMigrations.target_postgresql_version.to_i * 10000
```

Since there should only be changes to safety between major versions.

Reply:
Done. Updated the README as well.